src/Settings.php

```diff
@@ -150,6 +164,10 @@ public static function parseArguments(array $arguments)
                         $settings->excluded[] = $arguments->getNext();
                         break;
 
+                    case '-h':
```

Comment:
It will not work, because `-h` [is used for help](https://github.com/JakubOnderka/PHP-Parallel-Lint/blob/master/parallel-lint.php#L60)